spec/calendar.html

```diff
@@ -683,6 +705,46 @@ <h1>
         1. Return _merged_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-temporal-toisodayofyear" type="abstract operation">
+      <h1>
+        ToISODayOfYear (
+          _year_: an integer,
+          _month_: an integer,
+          _day_: an integer,
+        ): an integer
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It implements the logic for the `Temporal.Calendar.prototype.dayOfYear` method, which is used for the ISO 8601 calendar.</dd>
+      </dl>
+      <emu-alg>
+        1. Let _epochDays_ be MakeDay(year, month - 1, day).
+        1. Assert: _epochDays_ is finite.
+        1. Return DayWithinYear(MakeDate(_epochDays_, *+0*<sub>𝔽</sub>)) + *1*<sub>𝔽</sub>.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-toisodayofweek" type="abstract operation">
+      <h1>
+        ToISODayOfWeek (
+          _year_: an integer,
+          _month_: an integer,
+          _day_: an integer,
+        ): an integer
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It implements the logic for the `Temporal.Calendar.prototype.dayOfWeek` method, which is used for the ISO 8601 calendar.</dd>
+      </dl>
+      <emu-alg>
+        1. Let _epochDays_ be MakeDay(year, month - 1, day).
+        1. Assert: _epochDays_ is finite.
+        1. Let _dayOfWeek_ be WeekDay(MakeDate(_epochDays_, *+0*<sub>𝔽</sub>)).
+        1. If _dayOfWeek_ = *+0*<sub>𝔽</sub>, return *7*<sub>𝔽</sub>.
```

Comment:
can you elaborate on this line? is sunday the last day of the week or the first in ISO?